.github/workflows/winget-submission.yml

```diff
@@ -0,0 +1,27 @@
+name: Submit Microsoft.DevHome package to Windows Package Manager Community Repository
+
+on:
+  workflow_dispatch:
+  release:
+    types: [published]
+
+jobs:
+  winget:
+    name: Publish winget package
+    runs-on: windows-latest
+    steps:
+      - name: Submit package to Windows Package Manager Community Repository
+        run: |
+
+          $packageId = "Microsoft.DevHome"
+          $gitToken = "${{ secrets.WINGET_PAT }}"
+
+          # Fetching latest release from GitHub
+          $github = Invoke-RestMethod -uri "https://api.github.com/repos/microsoft/devhome/releases"
+          $targetRelease = $github | Where-Object -Property name -match 'Dev Home'| Select-Object -First 1
+          $installerUrl = $targetRelease | Select-Object -ExpandProperty assets -First 1 | Where-Object -Property name -match 'Windows.DevHome.*?msixbundle' | Select-Object -ExpandProperty browser_download_url
+          $packageVersion = $targetRelease.tag_name.Trim("v")
+
+          # Update package using wingetcreate
+          Invoke-WebRequest https://aka.ms/wingetcreate/latest -OutFile wingetcreate.exe
+          .\wingetcreate.exe update $packageId --version $packageVersion --urls "$installerUrl" --submit --token $gitToken
```

Comment:
Nit: Variables should enclosed in parentheses like `$(packageId)`. That's what we do for wingetcreate and the explanation is here: https://learn.microsoft.com/en-us/azure/devops/pipelines/process/variables?view=azure-devops&tabs=yaml%2Cbatch#macro-syntax-variables

Reply:
Prior to creating a PR here, I was testing this workflow in a private fork; It seems when I add $(packageId), it doesn't seem to work
![image](https://github.com/microsoft/devhome/assets/88161975/b5cef320-be7e-4ed7-aa44-d88f03437729)

I'll make the fork public if you want to see the details

Reply:
Ah that's interesting, I would imagine they would be similar but I guess not!